bitnami/suitecrm/templates/svc.yaml

```diff
@@ -28,7 +30,5 @@ spec:
       {{- if (and (eq .Values.service.type "NodePort") (not (empty .Values.service.nodePorts.https)))}}
       nodePort: {{ .Values.service.nodePorts.https }}
       {{- end }}
-  selector:
-    app: {{ template "suitecrm.name" . }}
-    release: "{{ .Release.Name }}"
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
+  publishNotReadyAddresses: true
```

Comment:
Why do you need `publishNotReadyAddresses: true`?

Reply:
The image is configuring SuiteCRM passing a wizard that will only work in the case of LoadBalancer services if the endpoint is reachable while the pod is not ready yet, since it is being configured.